src/LinkContainer.js

```diff
@@ -62,6 +62,10 @@ class LinkContainer extends React.Component {
       to, query, hash, state, children, onClick, target, action,
     } = this.props;
 
+    const { router } = this.context;
```

Comment:
`router` from context is used further down in the code but it's not being accessed via this variable (`this.context.router` instead of `router`).